docker/bash.sh

```diff
@@ -290,10 +289,10 @@ DOCKER_DEVICES=( )
 # If the user gave a shortcut defined in the Jenkinsfile, use it.
 EXPANDED_SHORTCUT=$(lookup_image_spec "${DOCKER_IMAGE_NAME}")
 if [ -n "${EXPANDED_SHORTCUT}" ]; then
-    if [ "${CI+x}" == "true" ]; then
+    if [ "${CI+x}" == "x" ]; then
```

Comment:
this seems to detect `CI` set to anything, rather than just `true`, is that intentional?

Reply:
with +x it merely evaluates to "x" if the var is set and nothing if not, iiuc